src/main/resources/scripts/anomalyDetectionTesting.txt

```diff
@@ -0,0 +1,79 @@
+##===========SETUP===============##
+sudo yum install -y java-17-amazon-corretto-devel
+java -version
+sudo yum -y update
+sudo yum -y install docker
+docker -v
+sudo service docker start
```

Comment:
Where are we stopping the docker process started here?

Reply:
Added the command for that as well now.